test/test_schema.rb

```diff
@@ -337,7 +365,7 @@ def self.resolve_type(_type, _obj, _ctx)
       end
     end
 
-    types = GraphQL::Client::Schema.generate(schema)
+    types = GraphQL::Client::Schema.generate(schema, raise_on_unknown_enum_value: true)
```

Comment:
Is this the default value? Could it be excluded here? (If so, I'd rather exclude it, so that we can continue testing the default behavior.)

```suggestion
    types = GraphQL::Client::Schema.generate(schema)
```